spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -260,39 +260,49 @@ def setup(self):
             text=_("Execute current line"),
             icon=self.create_icon('arrow-step-over'),
             triggered=lambda: self.debug_command("next"),
-            register_shortcut=True
+            register_shortcut=True,
+            context=Qt.ApplicationShortcut,
+            shortcut_context="_"
         )
 
         continue_action = self.create_action(
             DebuggerWidgetActions.Continue,
             text=_("Continue execution until next breakpoint"),
             icon=self.create_icon('arrow-continue'),
             triggered=lambda: self.debug_command("continue"),
-            register_shortcut=True
+            register_shortcut=True,
+            context=Qt.ApplicationShortcut,
+            shortcut_context="_"
         )
 
         step_action = self.create_action(
             DebuggerWidgetActions.Step,
             text=_("Step into function or method"),
             icon=self.create_icon('arrow-step-in'),
             triggered=lambda: self.debug_command("step"),
-            register_shortcut=True
+            register_shortcut=True,
+            context=Qt.ApplicationShortcut,
+            shortcut_context="debugger"
```

Comment:
Why `shortcut_context` is `debugger` for some actions and `_` for others? Shouldn't it be `_` for all?